### ISSUE TITLE
Switch from `:uuid` to `:elixir_uuid`

### ISSUE THREAD

**Issue by beorc** [commanded/commanded#179]
Apparently the `:uuid` library has changed it's name to `:elixir_uuid`. This causes a conflict error when building our app since commanded requires the old name:

```
==> Release failed, during .boot generation:
        Duplicated modules: 
    	'Elixir.UUID' specified in elixir_uuid and uuid
```

**Comment by beorc** [comment]
The `commanded-audit-middleware` library also has the `:uuid` dependency

**Comment by drozzy** [comment]
This is already in mix.exs, could we get a new relese @slashdotdash please?

**Comment by konstantinzolotarev** [comment]
And new error appears: 
```
Unchecked dependencies for environment dev:
* uuid (Hex package)
  could not find an app file at "_build/dev/lib/uuid/ebin/uuid.app". This may happen if the dependency was not yet compiled, or you specified the wrong application name in your deps, or the dependency indeed has no app file (then you can pass app: false as option)
** (Mix) Can't continue due to errors on dependencies
```

**Comment by slashdotdash** [comment]
Included in [v0.18.0](https://github.com/commanded/commanded/blob/master/CHANGELOG.md#v0180) and available from Hex.
